src/Components/PetContainer.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
-import '../Styles/PetContainer.scss';
 import PetCard from './PetCard';
+import '../Styles/PetContainer.scss';
 
 
-const PetContainer = () => {
+const PetContainer = ({pets}) => {
+  console.log(pets)
```

Comment:
This is testing that the props are working how they designed to.  And this will be replaced with logic to display the data